### PR TITLE
[MB-4782] Service member can click helper icons to read more about the different shipment types

### PR DESCRIPTION
**Pull request by SirenaBorracha**
## Description

On the Select Shipment Type page, the service member can click on a helper icon next to each of the two shipment options which opens a modal where the service member can read more about the HHG and PPM details. This behavior matches that of the helper icons that appear next to the NTS and NTSR options on the same page, which when clicked open a modal with more information about storage.

## Setup

```sh
make server_run
make client_run
```
Local login with an incomplete SM profile (I used needs@orde.rs) and follow through adding orders until you progress to the Select Move Type page. See that there is a helper icon to the right of the text in each of the two shipment cards. Click the helper icon in each of the shipment cards to see the helper text. Progress through, adding a move or a storage request, then click to add another shipment. On the select move type page, see that the helper icons are still there and clickable.

## Code Review Verification Steps

* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References 

* [Jira story](https://dp3.atlassian.net/browse/MB-4782) for this change

## Screenshots
I attempted to use Giphy Capture and was unable to scroll/interact with the page through the recording box. Will try harder next time.

**Added the helper icons:**
Screen Shot 2020-10-29 at 2 37 15 PM

**Once clicked:**
Screen Shot 2020-10-29 at 2 38 12 PM
Screen Shot 2020-10-29 at 2 38 26 PM

### CODE REVIEW

**Review by mariaclaratgolfo — APPROVED** [review]

File: src/components/Customer/modals/MoveInfoModal/MoveInfoModal.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      <li>Expert movers care for your things</li>
+      <li>Anything damaged in professional shipments will be replaced</li>
+    </ul>
+    <h5>Cons</h5>
+    <ul>
+      <li>Can only move on weekdays</li>
+      <li>May have to work around availability of movers</li>
+    </ul>
+    <h4>
+      <strong>PPMs: You get your things packed and moved, the government pays you</strong>
```
Comment (sojeri) [inline src/components/Customer/modals/MoveInfoModal/MoveInfoModal.jsx:29]: I'm surprised we're pluralizing PPM but not HHG

(no action necessary)

File: src/pages/MyMove/SelectMoveType.jsx [hunk 2]
```diff
@@ -114,6 +124,7 @@ export class SelectMoveType extends Component {
         cardText={hhgCardText}
         checked={moveType === SHIPMENT_OPTIONS.HHG}
         disabled={false}
+        onHelpClick={this.toggleMoveInfoModal}
       />
     );
     const hhgDisabledCard = (
```
Comment (sojeri) [inline src/pages/MyMove/SelectMoveType.jsx:130]: doesn't this one need the modal, too? I saw it got added to PPM disabled, so seems like it should be here as well.

File: src/pages/MyMove/SelectMoveType.test.jsx [hunk 3]
```diff
@@ -62,6 +63,29 @@ describe('SelectMoveType', () => {
       expect(wrapper.state('showStorageInfoModal')).toEqual(false);
       expect(storageInfoModal.prop('isOpen')).toEqual(false);
     });
+
+    it('renders the move info modal', () => {
+      expect(moveInfoModal.exists()).toBe(true);
+    });
+
+    it('the move info modal is closed by default', () => {
```
Comment (sojeri) [inline src/pages/MyMove/SelectMoveType.test.jsx:71]: nit: `"it the move info"` is a clunky read. if you want to call out that these tests relate to the move info modal, you should nest a `describe('the move info modal')` and group all these new tests under it. & either way IMO this particular case should be `it('is closed by default', ...);`

**Review by sojeri — COMMENTED** [review]
left a couple small comments & one off topic one relating to content. after I finish lunch, I'll pull your changes locally. wanted to get this feedback in first in case you'll update the code in the mean time. :)

**Review by suzubara — COMMENTED** [review]
+1 to @sojeri 's question about adding the modal to the disabled HHG card. Otherwise this looks great!

File: src/components/Customer/modals/MoveInfoModal/MoveInfoModal.jsx (hunk 1 above)
Comment (SirenaBorracha) [reply src/components/Customer/modals/MoveInfoModal/MoveInfoModal.jsx:29]: Checked with James, he requested it be changed. Thank you for catching this :)

File: src/pages/MyMove/SelectMoveType.jsx (hunk 2 above)
Comment (SirenaBorracha) [reply src/pages/MyMove/SelectMoveType.jsx:130]: Yes, again, thank you for catching this :)

**Review by sojeri — APPROVED** [review]
looking good! great work, @SirenaBorracha :)

looks like you need to accept the diff in happo, and then you can merge. :tada:

**Pull request merged**